app/models/user.rb

```diff
@@ -1,4 +1,6 @@
 class User < ActiveRecord::Base
+  attr_accessor :asserted_attributes
```

Comment:
SamlIdp expects an object that responds to `#asserted_attributes` and other methods  I think.